cli/src/main/java/dev/dirs/Directories.java

```diff
@@ -0,0 +1,70 @@
+package dev.dirs;
```

Comment:
I think we need to put this into it's distinct package, due to Java module system rules. `dev.dirs.cli` perhaps?

Reply:
Done.